### PR TITLE
Add redfish_info command to get service identification

### PR DESCRIPTION
**Pull request by stegosaur**
Proposed changes for https://github.com/ansible-collections/community.general/issues/7882

### CODE REVIEW

File: changelogs/fragments/7882-add-redfish-get-service-identification.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_command - Add redfish_info command to get service identification (https://github.com/ansible-collections/community.general/issues/7882)
```
Comment (felixfontein) [inline changelogs/fragments/7882-add-redfish-get-service-identification.yml:2]: The module this affects is `redfish_info`, not `redfish_command`, right?
```suggestion
  - redfish_info - add command ``GetServiceIdentification`` to get service identification (https://github.com/ansible-collections/community.general/issues/7882).
```

File: plugins/modules/redfish_info.py [hunk 2]
```diff
@@ -55,6 +55,11 @@
       - Security token for authenticating to OOB controller.
     type: str
     version_added: 2.3.0
+  manager:
+    description:
+      - Name of manager on OOB controller to target.
+    type: str
+    version_added: '8.2.0'
```
Comment (felixfontein) [inline plugins/modules/redfish_info.py:62]: 8.2.0 is already out, the next release will be 8.3.0:
```suggestion
    version_added: '8.3.0'
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

**Comment by mraineri** [comment]
Besides the documentation nits, everything in here looks good to me

**Comment by felixfontein** [comment]
@stegosaur thanks for contributing this!
@mraineri thanks for reviewing!

**Pull request merged**